SPECS/tdnf/tdnf.spec

```
@@ -72,9 +78,19 @@ Group:          Development/Libraries
 %description cli-libs
 Library providing cli libs for tdnf like clients.
 
+%package plugin-metalink
+Summary:        tdnf plugin providing metalink functionality for repo configurations
+Group:          Applications/RPM
+Requires:       tdnf = %{version}-%{release}
+Requires:       libxml2
+
+%description plugin-metalink
+tdnf plugin providing metalink functionality for repo configurations
+
 %package        plugin-repogpgcheck
 Summary:        tdnf plugin providing gpg verification for repository metadata
 Group:          Development/Libraries
+Requires:       tdnf = %{version}-%{release}
```

Similar comment regarding macro usage:
```suggestion
Requires:       %{name} = %{version}-%{release}
```